legistar/events.py

```diff
@@ -46,25 +42,38 @@ def events(self, follow_links=True, since=None) :
         # make sure we are not revisiting
         scraped_events = deque([], maxlen=10)
 
-        for page in self.eventPages(since) :
+        if since:
+            try:
+                since_year = since - 1
+            except TypeError:
+                raise TypeError('Expected integer for keyword arg :since, got "{since}" of type {type}'.format(since=since,
```

Comment:
I suppose this also needs to fail if `since` exceeds the current_year plus 1, eh?